source/guide_phpservermonitor.rst

```diff
@@ -0,0 +1,114 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: lang-php
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/phpservermonitor.png
```

Comment:
this file is missing

Reply:
How can I add the image to my pull request?

"Uploads are disabled.
File uploads require push access to this repository."

Reply:
Did you try in this repo (uberspace/lab)? It should work inside your own branch: https://github.com/thomas-kammerer/lab/tree/patch-8/source/_static/images